cves/kernel/CVE-2016-0823.yml

```diff
@@ -307,9 +316,11 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+  discussed_as_security: false
+  any_discussion: true
+  note: |
+    discussion on if the pagemap file should be completly unviewable by
```

Comment:
I couldn't find any actual discussion for this. But if you have links, you should add them!